lib/pos_mempool_transaction.go

```diff
@@ -0,0 +1,124 @@
+package lib
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/pkg/errors"
+	"time"
+)
+
+// MempoolTx contains a transaction along with additional metadata like the
+// fee and time added.
+type MempoolTx struct {
+	Tx *MsgDeSoTxn
+
+	// TxMeta is the transaction metadata
+	TxMeta *TransactionMetadata
```

Comment:
You're not setting this anywhere and it appears unused.

Reply:
Backwards-compatibility 😢